src/drivers/screen.c

```diff
@@ -0,0 +1,148 @@
+#include "screen.h"
+#include "../kernel/utils.h"
+
+#include "IO.h"
+
+void print_char(char character, int col, int row, char attribute_byte) {
+	uint8_t* vidmem = (uint8_t*) VIDEO_ADDRESS;
+
```

Comment:
Check vidmem for nullptr (same for all the pointers in this project.

If you dereference -> check them first. If you don't you will panic the entire kernel and your user will be super sad

Reply:
fixed, although do I really need to check it here? I know that it will always point to the same address since I defined it.

Reply:
It's just to be extra safe